files/build_templates/swss.service.j2

```diff
@@ -13,7 +13,8 @@ Before=ntp-config.service
 [Service]
 User=root
 Environment=sonic_asic_platform={{ sonic_asic_platform }}
-ExecStart=/usr/local/bin/swss.sh start
+ExecStartPre=/usr/local/bin/swss.sh start
+ExecStart=/usr/local/bin/swss.sh attach
```

Comment:
use wait

Reply:
Discussed offline that we should open a separate PR to change over to `docker wait`